stratum/hal/lib/common/utils.h

```diff
@@ -251,6 +252,9 @@ ::util::Status ConvertStringToLogSeverity(const std::string& severity_string,
 // Checks whether filename is a regular file and not a symlink
 bool IsRegularFile(const std::string& filename);
 
+// A helper method that constructs the IPsecNotification message
```

Comment:
"A helper method that" is just filler. Get to the point by starting with the verb. And say something more specific than "the ... message." For example, `Returns a text digest of an IPsecNotification message`, or `Summarizes an IPsecNotificationMessage.`